src/state/learnocaml_data.ml

```diff
@@ -711,7 +920,7 @@ module Exercise = struct
   module Index = struct
 
     type t =
-      | Exercises of (id * Meta.t option) list
+      | Exercises of (id * Meta.t option * Subindex.t option) list
```

Comment:
As the `Subindex.t` type already contains a `Meta.t`, I'd suggest refactoring this type into 3 constructors
(to avoid by construction the "case that can't occur" :)
Hoping this small refactoring won't have any impact on the rest!